[Test Merge] Merging tests CADMIN_1_22 and CADMIN_1_24 due to removal of test steps 2-4 in CADMIN_1_24

j-ororke opened the pull request:
Merge CADMIN_1_22 and CADMIN_1_24 python3 test modules:
- Removed test CADMIN_1_24
- Added test steps 5-6 for CADMIN_1_24 to test steps 7-8 for CADMIN_1_22 test module
- This change is a possible fix for task: [574](https://github.com/project-chip/matter-test-scripts/issues/574)

#### Testing
- Removing test CADMIN_1_24 to help with timing of test runs

j-ororke committed:
Merging tests CADMIN_1_22 and CADMIN_1_24 due to removal of timeout in test steps 2 and 3.
- Removed test CADMIN_1_24
- Added test steps 5-6 for CADMIN_1_24 to test steps 7-8 for CADMIN_1_22 test module
- This change is a possible fix for task: [574](https://github.com/project-chip/matter-test-scripts/issues/574)

j-ororke committed:
Updating TC_CADMIN_1_22 test module:
- Resolving linting issues

cecille commented on src/python_testing/TC_CADMIN_1_22.py:
```diff
@@ -74,6 +73,10 @@ def steps_TC_CADMIN_1_22(self) -> list[TestStep]:
                      "DUT_CE does not open its Commissioning window to allow a second commissioning. DUT_CE shows 'Failed to open commissioning window. Global status 0x85'"),
             TestStep(6, "TH_CR1 reads the window status to verify the DUT_CE window is closed",
                      "DUT_CE windows status shows the window is closed"),
+            TestStep(7, "TH_CR1 opens a commissioning window on DUT_CE using ECM with a value of 179 seconds",
```
If you're going to combine these tests (totally reasonable), also need a test to open for 180 and ensure it opens. You can use RevokeCommissioning to close it since we already test the autonomous timeout in another test.

j-ororke replied:
Updated test to include opening commissioning window for 180 seconds in test step 7 and revoking the open window in test step 8, moved prior test steps 7 and 8 to test steps 9 and 10.
Updated test plan PR #[5176](https://github.com/CHIP-Specifications/chip-test-plans/pull/5176) to match with updated test steps.

j-ororke committed:
Updating TC_CADMIN_1_22 python3 test module:
- Adding new test steps 7 and 8 for opening commissioning window for 180 seconds, then revoking commissioning window
- Moved prior test steps 7 and 8 to test steps 9 and 10

j-ororke committed:
Updating TC_CADMIN_1_22 python3 test module:
- Updated commissioning timeout in test step 7 to 180 seconds
- Updated test step order to be correct

j-ororke committed:
Merge branch 'master' into merge_CADMIN_1_22-24_tests

cecille committed:
Merge branch 'master' into merge_CADMIN_1_22-24_tests

restyled-commits committed:
Restyled by autopep8

cecille reviewed (changes requested):
Not actually requesting changes on the PR - I just want to block this from automerging before the test plan changes go in. If the test plan changes don't hit the *VE deadline, we should wait a week to merge this until the test harness is cut otherwise things will get weird.

cecille commented on src/python_testing/TC_CADMIN_1_22.py:
```diff
@@ -115,48 +122,17 @@ async def test_TC_CADMIN_1_22(self):
+        revokeCmd = Clusters.AdministratorCommissioning.Commands.RevokeCommissioning()
+        await self.th1.SendCommand(nodeid=self.dut_node_id, endpoint=0, payload=revokeCmd, timedRequestTimeoutMs=6000)
+        # The failsafe cleanup is scheduled after the command completes, so give it a bit of time to do that
+        sleep(1)
```
Does it work if you use the get_window_status function here rather than a hard-coded sleep?

j-ororke committed:
Updating TC_CADMIN_1_22 python3 test module:
- Changing from using a hard sleep timout to using get_window_status() to verify window is closed

restyled-commits committed:
Restyled by autopep8

j-ororke committed:
Updating CADMIN_1_22 python3 test module:
- Changed to using get_window_status() instead of hard sleep for test step 3

j-ororke committed:
Merge remote-tracking branch 'upstream/master' into merge_CADMIN_1_22-24_tests
